### DISCUSSION TITLE
socker_server.c中处理socket统计部分能加上每秒上下行流量统计？

### DISCUSSION THREAD

**Discussion by terry8210** [cloudwu/skynet#1760] [General]
socker_server.c中处理socket统计部分能加上每秒上下行流量统计？

**Comment by cloudwu** [comment]
这个模块不涉及时间，所以建议另写一个服务按时间取出流量数据换算。

**Reply by terry8210** [reply]
明白了，请问独立服务的采样时间范围和粒度怎么划分比较好？

**Reply by ericzhanchina** [reply]
用redis的zset，每10秒的时间作为member，流量做score，应该就可以

**Reply by terry8210** [reply]
谢谢你的建议，本来也是用10秒作为粒度，我觉得不需要借助redis，因为只是本节点的统计。

**Reply by ericzhanchina** [reply]
如果有多个节点的话，用redis就比较方便了
